cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/notifier/AlertaSelfHealingNotifier.java

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
Can we add documentation similar to `Configure-Slack-notifications.md` under `User Guide`?

Reply:
Done, I merged the 2 files